examples/chip-tool/py_matter_chip_tool_adapter/matter_chip_tool_adapter/encoder.py

```diff
@@ -16,7 +16,80 @@
 import json
 import re
 
+_ANY_COMMANDS_LIST = [
+    'ReadById',
+    'WriteById',
+    'SubscribeById',
+    'ReadEventById',
+    'SubscribeEventById',
+    'ReadAll',
+    'SubscribeAll',
+]
+
 _ALIASES = {
+    'AnyCommands': {
+        'alias': 'any',
+        'commands': {
+            'CommandById': {
+                'alias': 'command-by-id',
+                'arguments': {
+                    'ClusterId': 'cluster-id',
+                    'CommandId': 'command-id',
+                },
+            },
+            'ReadById': {
+                'alias': 'read-by-id',
+                'arguments': {
+                    'ClusterId': 'cluster-ids',
+                    'AttributeId': 'attribute-ids',
```

Comment:
Instead of making people do the 0xFFFFFFFF thing, could we make these optional arguments and allow just not passing them, then transcode that into the thing chip-tool expects under the hood?  I'd really rather we did not have the 0xFFFFFFFF thing spread if we can avoid it.

Reply:
Instead of making those optional arguments, I have updated the code to allow for "*" to be used as a wildcard. The adapter does translate that to the what `chip-tool`expects under the hood.